source/library/Signet/Unstable/Type/AsymmetricSignatureTest.hs

```diff
@@ -17,10 +19,10 @@ spec test = Test.describe test "Signet.Unstable.Type.AsymmetricSignature" $ do
 
     Test.it test "succeeds with valid input" $ do
       let result = AsymmetricSignature.parse $ Ascii.pack "QUJDREVGR0hJSktMTU5PUFFSU1RVVldYWVotMDEyMzQ1Njc4OS1hYmNkZWZnaGlqa2xtbm9wcXJzdHV2cXh5eg=="
-      signature <- Error.throwCryptoErrorIO . Ed25519.signature $ Ascii.pack "ABCDEFGHIJKLMNOPQRSTUVWXYZ-0123456789-abcdefghijklmnopqrstuvqxyz"
+      signature <- Either.throw . Error.eitherCryptoError . Ed25519.signature $ Ascii.pack "ABCDEFGHIJKLMNOPQRSTUVWXYZ-0123456789-abcdefghijklmnopqrstuvqxyz"
```

Comment:
It would be nice to have `throwCryptoError = Either.throw . Error.eitherCryptoError`, but I don't think it's worth the trouble to define.